examples/fluids/qfunctions/freestream_bc.h

```diff
@@ -395,8 +393,7 @@ CEED_QFUNCTION_HELPER int Freestream_Jacobian(void *ctx, CeedInt Q, const CeedSc
                                               RiemannFluxType flux_type) {
   const CeedScalar(*dq)[CEED_Q_VLA] = (const CeedScalar(*)[CEED_Q_VLA])in[0];
   const CeedScalar(*q_data_sur)     = in[2];
-  const CeedScalar(*x)[CEED_Q_VLA]  = (const CeedScalar(*)[CEED_Q_VLA])in[3];
-  const CeedScalar(*jac_data_sur)   = in[4];
+  const CeedScalar(*jac_data_sur)   = in[3];
```

Comment:
I'm guessing this is probably one of the sources of a segfault. The inputs to the qfunctions haven't been changed, so `jac_data_sur` is probably still assigned to `in[4]`.

Reply:
I did drop coordinates from the boundary integrals because their only need was for STATE.

Reply:
The STG case does run properly on both Polaris and on Sunspot.